plugins/pulp_rpm/plugins/distributors/export_distributor/generate_iso.py

```diff
@@ -87,6 +88,17 @@ def _make_iso(file_list, target_dir, output_dir, filename):
     if status != 0:
         log.error("Error creating iso %s; status code: %d; output: %s" % (file_path, status, out))
     else:
+        testFile = open(file_path, "rb")
```

Comment:
I would add a method called _add_digests_file(file_path) and then call it from here.

Reply:
testFile should probably be renamed to something more appropriate like exportedISO or ISOFile